utils/build-presets.ini

```diff
@@ -1386,6 +1386,33 @@ skip-test-swift
 skip-test-cmark
 skip-test-llbuild
 
+#===------------------------------------------------------------------------===#
+# Test swiftPM on Linux builder
+#===------------------------------------------------------------------------===#
+
+[preset: buildbot_swiftpm_linux_platform,tools=RA,stdlib=RA]
+mixin-preset=buildbot_incremental_base
+
+build-subdir=buildbot_incremental
+
+# Build Release without debug info, because it is faster to build.
+release
+assertions
+
+swiftpm
+
+xctest
+foundation
+libdispatch
+llbuild
+
+skip-test-swift
+skip-test-cmark
+skip-test-llbuild
+skip-test-libdispatch
+skip-test-foundation
+skip-test-xctest
```

Comment:
Are there other tests we should be skipping?

Reply:
Not that I know of.  The job currently passes: 

```
--skip-test-swift --skip-test-llbuild --skip-test-libdispatch --skip-test-cmark --skip-test-foundation --skip-test-xctest
```

which is the same, but reordered.

Reply:
NOTE: Eventually this will need to be changed so we run the integration tests. This is a good first step though!